src/Features.php

```diff
@@ -90,6 +92,15 @@ private static function get_fields(): array {
 				'id'   => self::THEME_EDITOR,
 				'type' => 'checkbox',
 			],
+			[
+				'name' => __( 'Allow beta blocks in post editor.', 'planet4-master-theme-backend' ),
+				'desc' => __(
+					'If enabled, beta blocks defined in <a href="https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/blob/master/planet4-gutenberg-blocks.php">this file</a>. Not for production use.',
+					'planet4-master-theme-backend'
```

Comment:
This is a bit confusing for editors/admins. The setting should completely hide the whole Beta category. We shouldn't link to the code for that. A more simple text would be:

```
'name' => 'Allow Beta Blocks in the editor'
'desc' => If enabled, blocks that are still not stable enough will be available.
```